README.md

```diff
@@ -1,78 +1,108 @@
+For interactionless use, set `ALLOWLISTED_PLUGINS=example-plugin1,example-plugin2,etc` in your `.env` file.
 
 ## Making a plugin
 
-Making a plugin is amazing! There's two routes to follow, first-party or third-party. First-party plugins are included in this repo and get included by default with all other plugins when the plugin platform is installed. Third-party plugins need to be added individually. Use first party plugins for plugins you expect others to use and want, and third party for things specifc to you. 
+Creating a plugin is a rewarding experience! You can choose between first-party or third-party plugins. First-party plugins are included in this repo and are installed by default along with other plugins when the plugin platform is installed. Third-party plugins need to be added individually. Use first-party plugins for plugins you expect others to use and want, and third-party for things specific to you.
 
 ### First Party How-To
-1. Clone the pluigins repo
-1. Follow the structure of the other plugins, implementing the plugin interface as required
-1. Write your tests
-1. Make a PR back to this repo!
+
+1. Clone the plugins repo
+2. Follow the structure of the other plugins, implementing the plugin interface as required
+3. Write your tests
+4. Make a PR back to this repo!
 
 ### Third Party How-To
+
 1. Clone [the third party template](https://github.com/Significant-Gravitas/Auto-GPT-Plugin-Template)
-1. Follow the instructions in the [third party template readme](https://github.com/Significant-Gravitas/Auto-GPT-Plugin-Template)
+2. Follow the instructions in the [third party template readme](https://github.com/Significant-Gravitas/Auto-GPT-Plugin-Template)
 
 ### Migrating Third Party to First Party
 
 > Thanks for contributing a plugin to the project!
 
 1. Clone this repo.
-1. Make a folder for your plugin under `src/autogpt_plugins`. Name it a simple desriptive name such as `notion`, `twitter` or `web_ui`.
-1. Take the files from your third party plugin located at `src/auto_gpt_plugin_template` and add them into the folder you created
-1. Add your readme from your third party plugin to the folder you created
-1. Add your plugin to the root readme with a description and a link to your plugin specific readme
-1. Add your plugin's python package requirements to `requirements.txt`
-1. Add tests to get your plugin to 80% code coverage
+2. Make a folder for your plugin under `src/autogpt_plugins`. Name it a simple descriptive name such as `notion`, `twitter`, or `web_ui`.
+3. Take the files from your third-party plugin located at `src/auto_gpt_plugin_template` and add them into the folder you created
+4. Add your readme from your third-party plugin to the folder you created
+5. Add your plugin to the root readme with a description and a link to your plugin-specific readme
+6. Add your plugin's Python package requirements to `requirements.txt`
+7. Add tests to get your plugin to 80% code coverage
```

Comment:
Change these back to 1. for each so they are autonumbered by the markdown tool

Reply:
done